ferret/explainers/__init__.py

```diff
@@ -27,6 +31,12 @@ def __init__(
                 "No helper provided. Using default 'text-classification' helper."
             )
             self.helper = create_helper(model, tokenizer, "text-classification")
+        else:
+            self.helper=model_helper
+        
+        # Initialize the correct helper in case the task at hand is NER.
+        if task_type == "token_classification":
+            self.helper = create_helper(model, tokenizer, "ner")
```

Comment:
Again, this is a fight against if-switching on top-level classes :) Can't we move this logic into the `create_helper` function?

Reply:
You are absolutely right, I just fixed that. We now introduced the possibility for the user to also specify the task_name parameter when initializing an explainer, e.g.
`l = LIMEExplainer(model, tokenizer, task_name="nli")`
This is not mandatory, but it is used to set the correct helper when working with explainers (otherwise the code would simply set the helper to the default SequenceClassifcationHelper, which is used only by text-classification and NLI). Alternatively the user could simply set the helper directly as an argument to the explainer, and that would work as well in the same way.